tests/suites/0_stateless/02_function/02_0014_function_maths.sql

```diff
@@ -43,7 +43,7 @@ SELECT '===exp===';
 SELECT exp(NULL);
 SELECT exp(2);
 SELECT exp('2');
-SELECT exp('a');
+SELECT exp('a'); -- {ErrorCode 1010}
```

Comment:
```
ERROR 1105 (HY000) at line 46: Code: 1010, displayText = Expected server error code: 1105 but got: 1010.
Cast error happens in casting from String to Float64.
```

I think that's appropriate.

Reply:
I think we should use the strict mode for casting by default, it's better than returning null or zero.